workspaces/services/azureml/readme.md

```diff
@@ -0,0 +1,37 @@
+# Azure machine Learning Serivce bundle
+
+See: [https://azure.microsoft.com/services/machine-learning/](https://azure.microsoft.com/services/machine-learning/)
+
+## Firewall Rules
+
+Please be aware that the following Firewall rules are opened for the workspace when this service is deployed:
+
+URLs:
+
+- graph.windows.net
+- ml.azure.com
+- login.microsoftonline.com
+- aadcdn.msftauth.net
+- graph.microsoft.com
+- management.azure.com
+- viennaglobal.azurecr.io
```

Comment:
I suggest describing why each of these are needed, as I would expect questions about this from a network administrator